.docker/main

```diff
@@ -1,3 +1,12 @@
+FROM python:3.8-buster as herdbook_api
+
+RUN mkdir -p code
+
+COPY app/ /code/
+WORKDIR /code
+
+RUN pip3 install -r /code/requirements-pydigree.txt && pip3 install -r /code/requirements.txt && chown -R www-data:www-data /code/
+
 FROM herdbook_api:latest
```

Comment:
With `:latest` here, this will be taken as an image reference and docker will thus try pulling it if it's not available locally.

But I think we can in fact  just remove the `as herdbook_api` in line 1 and the `FROM herdbook_api:latest`  (entire line 10).

Reply:
Yes that should work also. For me this separation was just confusing but if there is no need to have a api image stand alone it can just be one file!